### DISCUSSION TITLE
same image different engraving speed

### DISCUSSION THREAD

**Discussion by ronisegoly** [arkypita/LaserGRBL#1141] [Q&A]
Hi
I am looking for smart way to re engrave parts of imported image with different speed
I can of course erase parts of the image, making sure I keep the borders, so the SW does not change anything, but is there a smarter way?
![final to engrave](https://user-images.githubusercontent.com/5304613/96287190-36239e80-0fea-11eb-9cc1-fe661a232280.jpg)

**Comment by StuartB4** [comment]
You can cut out the part you need and save it as a seperate file, then after you have loaded the main image in click Append file and load it in and give it different speed and power settings.

![U676767676ntitled](https://user-images.githubusercontent.com/16980026/96304240-a7b41a80-0ff3-11eb-8641-f08762b46841.jpg)

![U7777676767676ntitled](https://user-images.githubusercontent.com/16980026/96304276-b8fd2700-0ff3-11eb-8318-29997474d243.jpg)

**Comment by ronisegoly** [comment]
I have played a bit, the best method for me now is when I use graphic SW which enables layers, to create basic layer with just 4 dots at each corner (to maintain size), and the then export to different images different  parts of the whole image, and then append on lasergrbl.
I engrave wood, so the only reason was to redo parts of the image for better results, without re engraving the whole image.
It opens new possibilities

**Comment by StuartB4** [comment]
Have you tried Lightburn?

**Comment by ronisegoly** [comment]
No, will look.
But it's not free

**Comment by StuartB4** [comment]
No it's a 30 day trial then it stops working unless you pay for a 12 month license. 
That costs $40 but you get updates for a year. 
Then if you don't want any more updates after the first year it keeps working as normal.
If you pay for another year it only costs $30 and you get the updates again.
I just payed for a second year and got 2 months extra free. So the new license is for 14 months. 
If you are going to use it a lot it is well worth it.

**Comment by StuartB4** [comment]
Also you can use the license on 2 computers, but if you need it on 3 just contact them and they will usually allow you to do that.
They will also extend the trial period if you speak to them near the end of it.
The help on various Lightburn Facebook groups is good, and the support from Lightburn is excellent.
Usually Oz, the developer will answer any queries / problems you might have using it, but there's a lot
of other people on there willing to help.

**Comment by ronisegoly** [comment]
I looked briefly.
What does it adds to what I have today? Able to import images to lasergrbl, and I have all the flexibility with graphics using my own tools

**Comment by StuartB4** [comment]
Too much for me to go in to detail here. If you have the time, have a look at some of the video tutorials, they will give you a better idea than i could here.
https://lightburnsoftware.com/pages/tutorials
